### PR TITLE
update coding style

### PR DESCRIPTION
**Pull request by mimmi20**
This PR wants to update the coding style. All surpressed warning in the coding style are fixed.

This PR is related to PR #1129.

### CODE REVIEW

File: ruleset.xml [hunk 1]
```diff
@@ -1,11 +1,20 @@
 <?xml version="1.0"?>
 <ruleset name="phing">
     <description>
-        PSR12 style with no namespace.
+        PSR2 style with no namespace.
```
Comment (siad007) [inline ruleset.xml]: Please use psr12 here

File: ruleset.xml [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
 
-    <rule ref="PSR12">
+    <!-- PHP keywords MUST be in lower case. -->
+    <rule ref="Generic.PHP.LowerCaseKeyword"/>
+
+    <rule ref="PSR2">
         <exclude name="PSR1.Classes.ClassDeclaration.MissingNamespace"/>
         <exclude name="PSR2.Methods.MethodDeclaration.Underscore"/>
+        <exclude name="PSR2.Classes.PropertyDeclaration.Underscore"/>
+        <exclude name="Generic.Files.LineLength.TooLong"/><!-- can not be suppressed -->
```
Comment (siad007) [inline ruleset.xml]: Could you remove the excludes? It just will hide all stuff which should be fixed instead.

File: ruleset.xml (hunk 1 above)
Comment (mimmi20) [reply ruleset.xml]: fixed

File: ruleset.xml (hunk 2 above)
Comment (mimmi20) [reply ruleset.xml]: fixed (expect the last 2, see the comment behind the excludes)

**Pull request merged**